### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Masterkenshin**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by PisethChhang** [comment]
OK

**Comment by Simplicityapparel** [comment]
Masterkenshin:patch-1

**Review by Simplicityapparel — COMMENTED** [review]
Masterkenshin:patch-1

**Review by Simplicityapparel — COMMENTED** [review]
Yes github

**Review by Simplicityapparel — COMMENTED** [review]
N

**Review by Simplicityapparel — COMMENTED** [review]
Acces

File: index.html [hunk 1]
```diff
@@ -2,3 +2,4 @@
 layout: index
 lang: en
 ---
```
Comment (PisethChhang) [inline index.html:4]: GPG key ID: 4AEE18F83AFDEB23

Comment (PisethChhang) [reply index.html:4]: TTDa6SdhUHcnYwLqYJ3iWYTAvwVtCgan26

Comment (brookreflections) [reply index.html:4]: Thanks
Sorry I dont quite understand
Regards Deb

**Pull request closed without merging**